application-helm/templates/create-mysql-app-db-user.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.application.persistance -}}
```

Comment:
This will be now `.Values.persistance` at all places

Reply:
Good catch

Reply:
Seems we already have persistence added to the values.yaml

```
persistence:
# -- Add Storage volumes(PVCs) to the pods
  enabled: false
  mountPVC: false
  mountPath: "/"
  accessMode: ReadWriteOnce
  ## If defined, storageClass: <storageClass>
  ## If set to "-", storageClass: "", which disables dynamic provisioning
  ## If undefined (the default) or set to null, no storageClass spec is
  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
  ##   GKE, AWS & OpenStack)
  ##
  storageClass: ""
  # Additional label added on PVC 
  labels:
    # key: "value"
  annotations:
  #  "helm.sh/resource-policy": keep
  storageSize: 8Gi
```

Reply:
So we would need to use a different name for it

Reply:
yeah, this was added if we needed pvc in any app...
maybe have `initdb` name